assignment1.txt

```diff
@@ -0,0 +1,23 @@
+Matthew Kellerman
+1.
+A compiler is a program that transforms one computer programming language   
+into another one. For example, you can use a compiler to change Python code into Assembly code.
+
+2.
+A compile error occurs when the code in a program is invalid. 
+A runtime error occurs when the code is correct, but it can't be carried 
```

Comment:
right, so compile time errors have more to do with the syntax, whereas run time has to do with the logic.